tests/system/test_system.py

```diff
@@ -1881,72 +1881,214 @@ def test_count_query_stream_empty_aggregation(query, database):
     assert "Aggregations can not be empty" in exc_info.value.message
 
 
-@firestore.transactional
-def create_in_transaction(collection_id, transaction, cleanup):
-    collection = client.collection(collection_id)
+@pytest.mark.parametrize("database", [None, FIRESTORE_OTHER_DB], indirect=True)
```

Comment:
the diff is a little confusing to view here. Were all the changes made for SUM/AVG? is it possible we can refactor some out into a separate PR?

Reply:
I changed `test_count_query_in_transaction` to be a more general, parameterized `test_aggregation_query_in_transaction` test. That also lead to some clean-up in the other query-in-transaction tests, since there were some (duplicated) helpers that were now only used in one place, and it made more sense to move the logic into that test itself. And I wanted to keep the related tests fairly consistent as well

Reply:
ok cool, thanks for the explanation!